### DISCUSSION TITLE
BLE GATTS and GATTC at the same time

### DISCUSSION THREAD

**Discussion by beazzles** [espressif/esp-at#796] [Q&A]
Is it possible to set up an ESP32-C3 as BLE Server and Client at the same time using the ESP-AT-Firmware? I tried to but if I set up the client first I receive an ERROR when I try to initialize the server and when I set up the server first all connections are disconnected when I try to initialize the client.

Thanks in advance.

**Comment by ustccw** [comment]
We are working on this and expect to support it soon. Once it's supported, I will let you know.

**Comment by ustccw** [comment]
@beazzles this feature is ready on master branch. we are testing related cases.
